src/dispatcher/index.js

```diff
@@ -0,0 +1,175 @@
+import amqp from 'amqplib';
+import { errorAndExit, log } from '../workerTemplate.js';
+
+const workerQueue = process.env.WORKERQUEUE;
+const resultQueue = process.env.RESULTSQUEUE;
+const rabbitHost = process.env.RABBITHOST;
+const rabbitUser = process.env.RABBITUSER;
+const rabbitPass = process.env.RABBITPASS;
+
+let channel;
+
+/**
+ * Listen for a job message and trigger the different involved workers.
+ * The entries in the job array are handled sequential, starting from top. The `type` of
+ * a worker configuration is used as the queue name to push the job and its configuration to.
+ * In the following example, the first worker will be called by passing a message to the
+ * queue named `download`, while the message itself is the original job message extend by
+ * a `nextJob` entry which indicates which task shall be handled next.
+ * If tasks finish succesfully, they usually return values which will then be appended to
+ * the specific task configuration via the `outputs` array.
+ *
+ * @example job message content:
+ * 
+  {
+    "job": [
+      {
+        "id": 123,
+        "type": "download-new-data-from-url",
+        "inputs": [
+          "https://deb.debian.org/debian/dists/bullseye/main/installer-amd64/current/images/cdrom/debian-cd_info.tar.gz",
+          "/home/data/"
+        ]
+      },
+      {
+        "id": 456,
+        "type": "gunzip",
+        "inputs": [
+          {
+            "outputOfId": 123,
+            "outputIndex": 0
+          }
+        ]
+      }
+    ]
+  }
+ */
+const dispatcher = async() => {
+  const connection = await amqp.connect({
+    hostname: rabbitHost,
+    username: rabbitUser,
+    password: rabbitPass,
+    heartbeat: 60
```

Comment:
we could make this configurable. but not urgent

Reply:
done